L1Trigger/L1TCaloLayer1/src/HLS_emulator.hh

```diff
@@ -0,0 +1,66 @@
+//The HLS4ML team has provided this interface to compiled model .so objects for the purpose of emulation
```

Comment:
This file looks like it should be in `plugins`, and have `.h` extension. Then the `#include` in `L1TCaloSummary.cc` could use just `"HLS_emulator.h"`.